### PR TITLE
[DPEDE-1717] Added infoIcon and infoIconMessage properties to Label component

### PR DESCRIPTION
**Pull request by wojciechwlodarczyklumen**
1. infoIcon property to Label component
2. infoIconMessage propertyto Label component
3. cypress tests
4. vue docs example

### CODE REVIEW

File: src/custom-elements/src/components/label/label.tsx [hunk 1]
```diff
@@ -38,7 +49,37 @@ export class Label {
...
 
+  componentWillLoad() {
+    this.helpButtonId = `help-button-${uuid4()}`;
+    this.helpPopoverId = `help-popover-${uuid4()}`;
+  }
+
+  componentDidLoad(): void {
+    this.el
+      .querySelector(`#${this.helpButtonId}`)
+      .addEventListener('click', () => (document.querySelector(`#${this.helpPopoverId}`) as any).toggle());
```
Comment (ASanchLumen) [inline src/custom-elements/src/components/label/label.tsx:60]: Could we use 

```suggestion
     <chi-button onChiClick="() => this.el.querySelector(`#${this.helpPopoverId}`) as any).toggle()"
```

in the render function? Traversing the DOM is usually slower

File: src/custom-elements/src/components/label/label.tsx [hunk 2]
```diff
@@ -38,7 +49,37 @@ export class Label {
...
+          variant="text"
+          arrow
+          reference={`#${this.helpButtonId}`}
+        >
+          {this.infoIconMessage}
+        </chi-popover>
+      </div>
+    ) : (
+      ''
+    );
```
Comment (ASanchLumen) [inline src/custom-elements/src/components/label/label.tsx:81]: I would move this to a function `this._getInfoIcon`, we can avoid complexity in render function that in the future is going to make SonarQube complain about high complexity functions.
```suggestion
    _getInfoIcon() {
      return this.infoIcon ? (
      <div class="chi-label__help">
        <chi-button id={this.helpButtonId} type="icon" size="xs" variant="flat" alternative-text="Help">
          <chi-icon icon="circle-info-outline"></chi-icon>
        </chi-button>
        <chi-popover
          id={this.helpPopoverId}
          position="top"
          variant="text"
          arrow
          reference={`#${this.helpButtonId}`}
        >
          {this.infoIconMessage}
        </chi-popover>
      </div>
    ) : (
      ''
    );
    }
    
    
    ....
    
    
    const infoIcon = this._getInfoIcon();
```

File: src/documentation/pages/components/label/examples/_help.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+</ComponentExample>
+</template>
+    
+    <script lang="ts">
+    import { Vue } from 'vue-facing-decorator';
+    
+    @NuxtComponent({
+      data: () => {
+        return {
+          exampleTabs: [
```
Comment (ASanchLumen) [inline src/documentation/pages/components/label/examples/_help.vue:25]: Please run prettier to fix indentation

**Comment by ASanchLumen** [comment]
Hi! Left some comments, also, visual tests are missing. Otherwise works as expected!

File: backstop-non-responsive.json [hunk 4]
```diff
@@ -1082,7 +1082,13 @@
...
         ".test-label-optional-md",
         ".test-label-optional-lg",
-        ".test-label-optional-xl"
+        ".test-label-optional-xl",
+        ".test-label-info-icon-message",
+        ".test-label-info-icon-message-xs",
+        ".test-label-info-icon-message-sm",
+        ".test-label-info-icon-message-md",
+        ".test-label-info-icon-message-lg",
+        ".test-label-info-icon-message-xl"
```
Comment (ASanchLumen) [inline backstop-non-responsive.json:1091]: The first selector includes the ones below, please remove the others and the pics as well.

```suggestion
        ".test-label-info-icon-message"
```

File: backstop-non-responsive-ce.json [hunk 5]
```diff
@@ -662,7 +662,13 @@
...
         ".test-label-optional-md",
         ".test-label-optional-lg",
-        ".test-label-optional-xl"
+        ".test-label-optional-xl",
+        ".test-label-info-icon-message",
+        ".test-label-info-icon-message-xs",
+        ".test-label-info-icon-message-sm",
+        ".test-label-info-icon-message-md",
+        ".test-label-info-icon-message-lg",
+        ".test-label-info-icon-message-xl"
```
Comment (ASanchLumen) [inline backstop-non-responsive-ce.json:671]:
```suggestion
        ".test-label-info-icon-message"
```

**Review by ASanchLumen — APPROVED** [review]
Approved with pending to remove the repeated visual tests

File: src/documentation/pages/components/label/examples/_help.vue [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        htmlblueprint: `<label class="chi-label">
+  Label
+  <div class="chi-label__help">
+    <chi-button id="example__help-button" type="icon" size="xs" variant="flat" alternative-text="Help">
+      <chi-icon icon="circle-info-outline"></chi-icon>
+    </chi-button>
+    <chi-popover id="example__help-popover" position="top" variant="text" arrow reference="#example__help-button">
+      Helpful information goes here.
+    </chi-popover>
+  </div>
```
Comment (ASanchLumen) [inline src/documentation/pages/components/label/examples/_help.vue:51]: We should use only HTML here right? @josecarlosgonzalezv

Comment (jllr) [reply src/documentation/pages/components/label/examples/_help.vue:51]: Correct @ASanchLumen. @wojciechwlodarczyklumen please change the html code snippets

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by jllr** [comment]
@wojciechwlodarczyklumen we have to bring back all the visual test that have been deleted for the different sizes.

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**